week-1.md

```diff
@@ -0,0 +1,83 @@
+
+Note that discussion of the details of proposals usually takes place outside of the PRs -- eg on the bitcoin-dev mailing list.
+
+It's probably a good idea to approach reviewing with a "constructive" mindset rather than a "judgemental" one -- that is coming up with ideas for how things might be changed to be better, or how things might not work as expected or hoped; rather than why particular approaches might be good or bad. In particular, Bitcoin might change in ways that even the smartest, most experienced Bitcoin developers think is a bad idea (or might not change in ways that they think is a good idea).
+
+One example of this might be the success of BIP 148, despite opposition by well respected developers and lack of support by the Bitcoin Core software:
+
+ * [BIP 148](https://github.com/bitcoin/bips/blob/master/bip-0148.mediawiki); List disccusion: [(1)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-March/013714.html) [(2)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-April/014152.html) [(3)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-May/014377.html); [PR#501](https://github.com/bitcoin/bips/pull/501); [Segwit Support page](https://en.bitcoin.it/wiki/Segwit_support); [Bitcoin Core merge NACK](https://github.com/bitcoin/bitcoin/pull/10442#issuecomment-305004154)
+
+So, personally, my recommendation is to focus almost all of your review effort on understanding the proposal and finding ways to make it work better, and not about whether you'll ultimately end up "for" or "against" it. That way your review effort will make things better whatever ends up happening. That doesn't mean you shouldn't form an opinion as to whether the change is a good one or not, of course, just try not to make that your primary focus!
+
+## Taproot
+
+The current draft of the taproot BIP can be found at:
+
+ * https://github.com/sipa/bips/blob/bip-schnorr/bip-taproot.mediawiki
+
+Remember that this is being actively worked on, so there may have been changes if you've read it previously!
+
+We'll be spending more time later on how "script path" spends work and the details of how signatures work, so you may wish to skip or skim the sections titled "script path spending", "Signature validation rules", and "Constructing and spending Taproot outputs" for now.
+
+Ideally reading the BIP (and its references) will already make it clear what the purpose of the proposal is; and how it achieves that purpose. Some good general questions to think about might include:
```

Comment:
The semicolon in this sentence seems out of place, I'd suggest either omitting it or converting it into an emdash.